deploy/role_binding.yaml

```diff
@@ -10,3 +10,16 @@ roleRef:
   kind: ClusterRole
   name: unifiedpush-operator
   apiGroup: rbac.authorization.k8s.io
+---
+kind: ClusterRoleBinding
```

Comment:
Could you explain what's going on with this ClusterRoleBinding, it seems odd that we'd be adding a cluster role to a group that isn't our operator

Reply:
I copied it from Apicurio operator. 

@odra any idea?
@grdryn sounds right.

Reply:
I deleted this one and things continued to work.
Probably a left over when they wanted to test image tagging on OpenShift.
@odra  FYI